src/ai/backend/manager/api/vfolder.py

```diff
@@ -646,7 +646,7 @@ async def list_folders(request: web.Request, params: Any) -> web.Response:
     ),
 )
 async def delete_by_id(request: web.Request, params: Any) -> web.Response:
-    await ensure_vfolder_status(request, VFolderAccessStatus.DELETABLE, params["id"])
+    await ensure_vfolder_status(request, VFolderAccessStatus.DELETABLE, uuid.UUID(params["id"]))
```

Comment:
To get rid of similar human errors in the future, I guess it is better to enforce the type of "id" parameter as `tx.UUID` (instead of current plaintext-ish `t.String` validator). In that way, `params["id"]` itself will be delivered as a form of `UUID()` type instance.

Reply:
I was worried about those which already are using this API. If it is okay, I agree that it would be better to change the type to tx.UUID.